src/pages/home/sidebar/SidebarScreen/FloatingActionButtonAndPopover.js

```diff
@@ -81,7 +83,12 @@ class FloatingActionButtonAndPopover extends React.Component {
     }
 
     componentDidMount() {
-        const routes = lodashGet(this.props.navigation.getState(), 'routes', []);
+        const navigationState = this.props.navigation.getState();
+        const currentRoute = navigationState.routes[navigationState.index];
```

Comment:
It's better we still use `routes` from the `lodashGet` to ensure the `routes` is not undefined and avoid error `Cannot read properties of undefined`.